static/src/javascripts/projects/common/modules/commercial/creatives/gu-style-comcontent.js

```diff
@@ -0,0 +1,51 @@
+define([
+    'fastdom',
+    'common/utils/$',
+    'common/utils/_',
+    'common/utils/detect',
+    'common/utils/mediator',
+    'common/utils/template',
+    'common/views/svgs',
+    'common/modules/commercial/gustyle/gustyle',
+    'text!common/views/commercial/creatives/gu-style-comcontent.html'
+], function (
+    fastdom,
+    $,
+    _,
+    detect,
+    mediator,
+    template,
+    svgs,
+    GuStyle,
+    gustyleComcontentTpl
+) {
+
+    var GustyleComcontent = function ($adSlot, params) {
+        this.$adSlot = $adSlot;
+        this.params  = params;
+    };
+
+    GustyleComcontent.prototype.create = function () {
+        var externalLinkIcon = svgs('externalLink', ['gu-external-icon']),
+            templateOptions = {
```

Comment:
Maybe we should use the terminology 'model' or 'viewmodel' for objects we want templating?